.github/pull_request_template.md

```diff
@@ -1,53 +1,98 @@
 # Description
 
-<!-- Clear description what this PR does and why it's needed -->
+Provide a clear and concise description of the changes in this PR and
+explain why they are necessary.
 
-<!-- For Backport PRs, please note the following:
+If the PR contains only one commit, you will see the commit message above:
+fill free to use it under the description section here, if it is good enough.
 
-- Add a note to indicate the origin of the fix, for example:
+For **Backport PRs**, a full description is optional, but please clearly state
+the original PR number(s). Use the #{NUMBER} format for that, it makes it easier
+to handle with the scripts later. For example:
 
-Backport of #<original-PR-number>
+> Backport of #1234, #5678, #91011
 
-- PR's title should also indicate the stable branch, for instance:
+Description of a backport PR must also follow the following format:
 
-[<stable-branch>] Original's PR title
--->
+> "[x.y-stable] Original PR title".
+
+where `x.y-stable` is the name of the target stable branch, and
+`Original PR title` is the title of the original PR.
+
+For example, for a PR that backports a PR with title `Fix the nasty bug` to
+branch `13.4-stable` the title should be:
+`[13.4-stable] Fix the nasty bug`.
 
 ## PR dependencies
 
-<!-- List all dependencies of this PR (when applicable) -->
+List all dependencies of this PR (when applicable, otherwise remove this
+section).
 
 ## How to test and validate this PR
 
-<!-- Please describe how the changes in this PR can be validated or
-verified. For example:
+Please describe how the changes in this PR can be validated or verified. For
+example:
 
 - If your PR fixes a bug, outline the steps to confirm the issue is resolved.
 - If your PR introduces a new feature, explain how to test and validate it.
--->
+
+This will be used
+
+1. to provide test scenarios for the QA team
+1. by a reviewer to validate the changes in this PR.
+
+The first is especially important, so, please make sure to provide as much
+detail as possible.
+
+If it's covered by an automated test, please mention it here.
 
 ## Changelog notes
 
-<!-- Short description to be included in the ChangeLog notes -->
+Text in this section will be used to generate the changelog entry for
+release notes. The consumer of this - is end users, not developers.
+So, provide a clear and short description of what is changes in the PR from
+the end user perspective. If it changes only tooling or some internal
+implementation, you can skip this section.
```

Comment:
In case of changes that do not require Chagelog info, is it safe to just skip it (removed it), or maybe explicitly indicate that, e.g., with something like `N/A`, `Not required`? I prefer that later....

Reply:
Yep, makes sense